### PR TITLE
feat: allow to set custom arguments for package manager

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
feature

**Did you add tests for your changes?**
Yes
**If relevant, did you update the documentation?**
No
**Summary**
Allow passing more arguments to the package manager.

**Does this PR introduce a breaking change?**
Yes, `options.quiet` was removed in favor of `packageManager.options.quiet`

**Other information**
No

### CODE REVIEW

File: test/__snapshots__/plugin.test.js.snap [hunk 1]
```diff
@@ -34,7 +33,6 @@ Array [
     },
     "packageManager": "npm",
     "prompt": true,
-    "quiet": false,
```
Comment (alexander-akait) [inline test/__snapshots__/plugin.test.js.snap:37]: I will look at other repos where this plugin are using, I think better to keep `quiet` for better DX, let's move it in `packageManager.options.quiet` (just put `--silent`/`'--silent', '--no-progress'`) in arguments, also let's add pnpm support

File: src/installer.js [hunk 2]
```diff
@@ -290,13 +293,15 @@ module.exports.install = async function install(deps, options, logger) {
       packageManager.options && packageManager.options.dev
         ? '--save-dev'
         : null;
+    quietOptions = [];
```
Comment (alexander-akait) [inline src/installer.js]: it should be `--reporter=silent` https://pnpm.io/cli/install#--reportername

File: test/__snapshots__/plugin.test.js.snap (hunk 1 above)
Comment (snitin315) [reply test/__snapshots__/plugin.test.js.snap:37]: Updated. It seems `pnpm` don't have `--silent` flag for the [`add` comman](https://pnpm.io/cli/add#options).

Comment (alexander-akait) [reply test/__snapshots__/plugin.test.js.snap:37]: some options are global, some only for commands, `--reporter=silent` should be global you can test it locally

File: src/installer.js (hunk 2 above)
Comment (snitin315) [reply src/installer.js]: updated

**Review by alexander-akait — APPROVED** [review]
@snitin315 What is the next?

**Comment by snitin315** [comment]
@alexander-akait I think all major tasks are done. I will update docs next.

**Comment by alexander-akait** [comment]
@snitin315 Yep, let's do it, I think time to do release

**Pull request merged**